lib/susedistribution.pm

```diff
@@ -715,7 +715,8 @@ sub activate_console {
             # case the system is still booting (https://bugzilla.novell.com/show_bug.cgi?id=895602)
             # or when using remote consoles which can take some seconds, e.g.
             # just after ssh login
-            wait_still_screen 1;    # Wait a bit to avoid false match on 'text-logged-in-$user', if tty has not switched yet
+            # Wait a bit to avoid false match on 'text-logged-in-$user', if tty has not switched yet
+            wait_still_screen 6, 180;
```

Comment:
The `wait_still_screen` call has been introduced with bed94ef77 and I still consider it wrong. The task is to just create unambiguous needles and only match on the right screen after we switched to the right tty. If you see any premature matches then the problem is most likely in ill-designed needles.

Reply:
Hi Oliver, this issue will cause some migration test random fail. The solution to wait more time is simple and may not the final solution but it really works (see so many verify log attached) for this issue. (In fact, from my view, the correct fix should change to the wait screen to wait a signal that the switch to tty is ready). 
Could we merge this PR as a workaround for this issue? And file a new PR to fix the root cause. WDYT?

Reply:
Sorry, no.